CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.0.0 (26.01.2022)
+
+Removed [pngquant-bin](https://github.com/imagemin/pngquant-bin) due to license issues.
```

Comment:
Let's add a simple explanation into migration guide? Now as a user I don't understand what should I expect after this major update. It would be fair to highlight the issues users might occur after updating to 3.0.0.